code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -50,6 +63,20 @@
 	broadcast() // Bzzt
 
 /obj/machinery/media/transmitter/broadcast/attackby(var/obj/item/W, mob/user)
+	if(panel_open && (istype(W, /obj/item/device/multitool)||istype(W, /obj/item/weapon/wirecutters)))
+		attack_hand(user)
+	if(issolder(W))
```

Comment:
Nanopaste repairs electronic components, the only reason it is "associated with robots" is that it hasn't been given many other purposes yet. I could say the same with your solder being associated with circuitboards and thus being a strange choice.